### PR TITLE
Extender creative fix

### PR DESCRIPTION
**Pull request by 256p**
Added super.onStart call
Also added missing bracket in CSS
According to specs, creative must answer on startCreative message to allow player to display it. https://interactiveadvertisingbureau.github.io/SIMID/#workflow-startCreative

### CODE REVIEW

**Review by ryanthompson591 — APPROVED** [review]
Hey, thanks for the fix. Feel free to merge this if you are satisfied!

**Comment by 256p** [comment]
Hi, I don't have write access, so someone who does need to merge this

**Pull request merged**
